Library/Formula/vim.rb

```diff
@@ -74,11 +74,6 @@ def install
       opts << "--enable-luainterp"
     end
 
-    # XXX: Please do not submit a pull request that hardcodes the path
-    # to ruby: vim can be compiled against 1.8.x or 1.9.3-p385 and up.
-    # If you have problems with vim because of ruby, ensure a compatible
-    # version is first in your PATH when building vim.
```

Comment:
The Ruby dependency handles it. It'll build bottles against Homebrew's Ruby (I haven't decided whether to actually pull a bottle on this yet) but if you build it from source it'll build against the first valid Ruby it finds in the `$PATH`. The inclusion of a `depends_on :ruby` should make it obvious enough hardcoding would be invalid, theoretically.